tests/functional-tests/README.md

```diff
@@ -0,0 +1,77 @@
+# IBM Spectrum Scale CSI - Automation
+
+The IBM Spectrum Scale Container Storage Interface (CSI) project enables container orchestrators, such as Kubernetes and OpenShift, to manage the life-cycle of persistent storage.
+
+This project contains an automation framework to exercise the IBM Spectrum Scale CSI Driver and Operator functionality .
+
+
+### Testbed environment 
+
+- IBM Spectrum Scale Cluster - 5.0.4.1+ Version  (**IBM Spectrum Scale supported kernel version**)
+- Kubernetes Cluster Version 1.14 - 1.18
+- Openshift Version 4.2.x,4.3.x
+- IBM Spectrum Scale Cluster CSI Version - 1.1.0+
+
+### Pre-requesite for automation framework
+
+Install Python (3.7.4 or higher) and below mentioned pip modules:
+
+  ``` 
+        python3.7 -m pip install kubernetes
+        python3.7 -m pip install pytest
+        python3.7 -m pip install pytest-html
+        python3.7 -m pip install jsmin
```

Comment:
Are all these spaces  in front required? This adds a lot of extra spacing when users copy/paste

```
python3.7 -m pip install kubernetes
python3.7 -m pip install pytest
python3.7 -m pip install pytest-html
python3.7 -m pip install jsmin
```